lib/puppet-languageserver/message_router.rb

```diff
@@ -57,12 +57,13 @@ def receive_request(request)
 
       when 'puppet/getVersion'
         request.reply_result(LSP::PuppetVersion.new(
-                               'puppetVersion'   => Puppet.version,
-                               'facterVersion'   => Facter.version,
-                               'factsLoaded'     => PuppetLanguageServer::FacterHelper.facts_loaded?,
-                               'functionsLoaded' => PuppetLanguageServer::PuppetHelper.default_functions_loaded?,
-                               'typesLoaded'     => PuppetLanguageServer::PuppetHelper.default_types_loaded?,
-                               'classesLoaded'   => PuppetLanguageServer::PuppetHelper.default_classes_loaded?
+                               'languageServerVersion' => PuppetEditorServices.version,
```

Comment:
does this mean the client has to have this interface? i don't think langserverversion is there